src/components/Form.js

```diff
@@ -0,0 +1,326 @@
+
+
+const Form = (props) => {
+
+    return(
+        <form onSubmit={props.handleSubmit}>
+
+            <legend>So... You love podcasts and you want to discover more? Great! Here is the #1 app to get some recommendations. Enter your keyword, and click on Search Podcast button! You may select country or genre to help with the search results. Enjoy!</legend>
+
```

Comment:
Hi, I'm see the label, mas I'm not seen the text inside the label tag. it could be hard to screen readers